amazon-msk/cdk/lib/templates/SecureAccess/zilla.yaml.mustache

```diff
@@ -93,4 +102,3 @@ bindings:
     routes:
```

Comment:
We can simplify the tcp_client options like we did for confluent cloud SecureAccess